### ISSUE TITLE
Disabling a linter doesn't work consistently

### ISSUE THREAD

**Issue by matt-bailey** [causes/scss-lint#377]
I have three files that contain vendor prefixes which I'd like scss-lint to ignore, so I've added `// scss-lint:disable VendorPrefixes` to the top of each one.

This works fine for one of the files, but the other two still throw warnings no matter where I put the `disable` in the file.

Warning example:

```bash
sass/tools/_gradients.scss
  17: [W] VendorPrefixes:  Avoid vendor prefixes.
```

The file that 'works' contains flexbox code like this:

```scss
%grid--flexbox {
    @include rem(margin-left, -$base--spacing-unit);
    display: -webkit-box; /* OLD - iOS 6-, Safari 3.1-6 */
    display: -moz-box; /* OLD - Firefox 19- (buggy but mostly works) */
    display: -ms-flexbox; // TWEENER - IE 10
    display: -moz-flex;
    display: -webkit-flex; // NEW - Chrome
    display: flex; // NEW, Spec - Opera 12.1, Firefox 20+

    -ms-flex-wrap: wrap;
    -webkit-flex-wrap: wrap;
    flex-wrap: wrap;
}
```

One of the files that doesn't 'work' only contains an opacity mixin:

```scss
@mixin opacity($value, $important: false) {
    $ie-value: $value * 100;
    @if ($important) {
        opacity: $value !important;
    } @else {
        opacity: $value;
    }
    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity='+$ie-value+')";
    filter: alpha(opacity=$ie-value);
}
```

The other file that doesn't 'work' contains a gradient mixin:

```scss
@mixin linear-gradient($start-color, $stop-color, $fallback-color) {
    background-color: $fallback-color; /* Old browsers */
    background-image: -moz-linear-gradient(top, $start-color 0%, $stop-color 100%); /* FF3.6+ */
    background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0%, $start-color), color-stop(100%, $stop-color)); /* Chrome,Safari4+ */
    background-image: -webkit-linear-gradient(top, $start-color 0%, $stop-color 100%); /* Chrome10+,Safari5.1+ */
    background-image: -o-linear-gradient(top, $start-color 0%, $stop-color 100%); /* Opera 11.10+ */
    background-image: -ms-linear-gradient(top, $start-color 0%, $stop-color 100%); /* IE10+ */
    background-image: linear-gradient(to bottom, $start-color 0%, $stop-color 100%); /* W3C */
}
```

Can you see any reason why disabling the linter in these two cases would still trigger the warning?

I can, of course, add these two files to the 'exclude' list (I'm using the [scss-lint grunt task](https://github.com/ahmednuaman/grunt-scss-lint)), but it would be nice to have things work in a consistent way.

**Comment by sds** [comment]
Hey @matt-bailey,

I can't reproduce the issue with the latest version of `scss-lint` (0.34.0 at time of writing). I believe this bug was fixed in #355.

**Comment by matt-bailey** [comment]
Thanks @sds, I'll check the version I have installed.
